caravel/assets/javascripts/SqlLab/components/SqlEditorLeft.jsx

```diff
@@ -102,22 +102,27 @@ class SqlEditorTopToolbar extends React.Component {
     const tableName = tableOpt.value;
     const qe = this.props.queryEditor;
     const url = `/caravel/table/${qe.dbId}/${tableName}/${qe.schema}/`;
+
+    this.setState({ tableLoading: true });
```

Comment:
`state.tableLoading` is used at line 164. It's the Select's inline loading spinner. I used it only for when the content was loading before, now I also use it while the table metadata is loading.